packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -99,3 +99,96 @@ export const createCustomFocusIndicatorStyle = (
   },
   [`${KEYBOARD_NAV_SELECTOR} :${options.selector || defaultOptions.selector}`]: style,
 });
+
+/**
+ * @param pressedBorderColor - color used in case user presses again the field.
+ * @param targetChild - primary element if style is applied to a wrapper.
+ */
+export type BottomFocusIndicatorOptions = CreateFocusIndicatorStyleRuleOptions & {
+  borderWidth: string;
+  borderRadius: string;
+  borderColor: string;
+  pressedBorderColor: string;
+  targetChild?: keyof JSX.IntrinsicElements;
+};
+
+/**
+ * Creates a bottom border focus indicator used in components such as Input and TextArea.
+ * NOTE: This style uses border instead of outline
+ *
+ * @param options - Configure the style of the focus indicator
+ */
+export const getBottomFocusIndicator = (options: BottomFocusIndicatorOptions): GriffelStyle => {
+  const {
+    selector = 'focus-within',
+    borderWidth,
+    borderRadius,
+    borderColor,
+    pressedBorderColor,
+    targetChild,
+  } = options;
+
+  // disable default browser outline if `focus-within` selector is used.
+  const outlineStyle =
+    selector === 'focus-within'
+      ? {
+          [`& > ${targetChild}`]: {
+            ':focus-visible': {
+              outlineStyle: 'none',
```

Comment:
since you're removing the native outline: have you tested this focus outline in windows high contrast mode ?

Reply:
I think that would be covered below, in this block (i.e. it seems like the outline is removed from the child, but added to the parent):
```
[`:${selector}`]: {
      outlineWidth: borderWidth,
      outlineStyle: 'solid',
      outlineColor: 'transparent',
},
```

Reply:
The outline does show up in windows high contrast mode :/ I will have to see how we could remove it since it overlaps the custom focus indicator.